Dockerfile

```diff
@@ -0,0 +1,31 @@
+# syntax = docker/dockerfile:experimental
+FROM golang:1.15 as builder
+
+ENV GRPC_HEALTH_PROBE_VERSION="v0.3.1"
+RUN wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
```

Comment:
the kubelet will literally run this binary to perform health checks, so it needs to be fetched here and copied to the final image.